src/_variables.scss

```diff
@@ -357,6 +357,7 @@ $menu-fade-duration: 0.2s !default;
 $list-container-height: 500px !default;
 $list-container-width: 400px !default;
 $list-container-item-height: 70px !default;
+$list__item-hover-color: #eeeeee !default;
```

Comment:
Hardcoded colors like this should be avoided. I believe `gray-300` from our color palette would be a better choice here.